nearcore/src/config.rs

```diff
@@ -431,6 +432,7 @@ pub struct Config {
     /// If set, overrides value in genesis configuration.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub max_gas_burnt_view: Option<Gas>,
+    pub num_epochs_to_keep_store_data: u64,
```

Comment:
Please add a default value for serde, similar to what we did for other fields

Reply:
Ok, I figured it out and updated the PR.